process.go

```diff
@@ -245,7 +245,19 @@ func processImage(ctx context.Context) ([]byte, error) {
 	checkTimeout(ctx)
 
 	imgWidth, imgHeight, angle, flip := extractMeta(img)
+	if po.Dpr > 1 {
+		if po.Width != 0 {
+			po.Width = int(float32(po.Width) * po.Dpr)
+		} else {
+			po.Width = int(float32(imgWidth) * po.Dpr)
```

Comment:
This breaks the behavior. Zero width/height should stay zero.

Reply:
Thanks for the feedback, but what about when the users do not request any width/height in specific, and he just wants to get an image with dpr=2 for instance. where is the right location to apply this?